### PR TITLE
feat(api): add string ids option to contact get data endpoint

### PR DESCRIPTION
**Pull request by gabor-balla-emarsys**

### CODE REVIEW

**Comment by adambrunner** [comment]
I think that these are not `stringId`s, even if the Suite API calls them that way. I'd call this option `nameAsKey` or something like it.

File: api/endpoints/contact/index.js [hunk 1]
```diff
@@ -52,9 +56,11 @@ _.extend(Contact.prototype, {
     return this._requireParameters(payload, ['keyValues']).then(function() {
       logger.log('contact_getdata');
 
+      var path = this._useStringIds(options) ? '/contact/getdata/stringids=1' : '/contact/getdata';
+
       return this._request.post(
         this._getCustomerId(options),
-        '/contact/getdata',
+        path,
```
Comment (adambrunner) [inline api/endpoints/contact/index.js]: Please rename `path` to `url`.

**Comment by gabor-balla-emarsys** [comment]
I'm not sure about the right wording, but I would stay consistent with the naming convention used by the API / API documentation.

File: api/endpoints/contact/index.js (hunk 1 above)
Comment (gabor-balla-emarsys) [inline api/endpoints/contact/index.js]: Done.

**Comment by adambrunner** [comment]
LGTM

**Comment by gabor-balla-emarsys** [comment]
Thank you!

**Pull request merged**